app/meta_development.go

```diff
@@ -0,0 +1,40 @@
+//go:build !no_metadata
+
+package app
+
+import (
+	"os"
+	"path/filepath"
+
+	"fyne.io/fyne/v2"
+	"fyne.io/fyne/v2/internal/metadata"
+)
+
+func checkLocalMetadata() {
+	file, _ := filepath.Abs("FyneApp.toml")
```

Comment:
You might want to check the error here, I think

Reply:
This went away